<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDlgClass</class>
 <widget class="QWidget" name="ChatDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frmChat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>260</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QGroupBox" name="groupChatTop">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>240</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(67, 67, 67);
border:0px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="lblChatClient">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>1</y>
       <width>235</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: rgb(42, 42, 42);
background-color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>Chat </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnChatMaxium">
     <property name="geometry">
      <rect>
       <x>217</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: rgba(255, 255, 255,0);
border-color: rgb(0, 0, 0);
border:2px
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Resources/max.png</normaloff>:/Resources/max.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnChatMinium">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: rgba(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Resources/min.png</normaloff>:/Resources/min.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblChatMark">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: rgba(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupChatContent">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>24</y>
      <width>235</width>
      <height>235</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border:0px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="messageView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>235</width>
       <height>233</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>color: rgb(33, 33, 33); 
background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="edtCustomerMessage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>92</y>
       <width>201</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(127, 127, 127, 255)); 
border-radius:5px;
color: rgb(0, 0, 127);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="edtMyMessage">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>135</y>
       <width>201</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(127, 127, 127, 255)); 
border-radius:5px;
color: rgb(0, 0, 127);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupMessageEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>181</y>
       <width>236</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: rgb(226, 226, 226);
border:0px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QPlainTextEdit" name="edtMessage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>215</width>
        <height>49</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSend">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>14</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Resources/arrow.png</normaloff>:/Resources/arrow.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>btnSend</zorder>
     <zorder>edtMessage</zorder>
    </widget>
    <widget class="QPushButton" name="btnReboot_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>235</width>
       <height>6</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:0.506, stop:0 rgba(20, 20, 20, 255), stop:0.920455 rgba(255, 255, 255, 255));
border:1px solid #6c6c6c;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
